src/tree/CosmosDBAccountWizard/CosmosDBAccountCapacityStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../utils/localize';
+import { ICosmosDBWizardContext } from './ICosmosDBWizardContext';
+
+export class CosmosDBAccountCapacityStep extends AzureWizardPromptStep<ICosmosDBWizardContext> {
+
+    public async prompt(wizardContext: ICosmosDBWizardContext): Promise<void> {
+
+        const placeHolder: string = localize('selectDBServerMsg', 'Select a capacity model.')
+
+        wizardContext.isServerless = (await ext.ui.showQuickPick(
+            [
+                { label: localize('provisionedOption', 'Provisioned Throughput'), data: false },
+                { label: localize('serverlessOption', 'Serverless'), data: true }
+            ], { placeHolder })).data;
```

Comment:
It might be worth adding a "Learn More" link here.  I know the portal uses the same verbage, but provisioned throughput and serverless feel like they don't mean a whole lot to me.

The portal uses this one: https://aka.ms/cosmos-models

Reply:
You can do it like this: https://github.com/microsoft/vscode-azurestaticwebapps/blob/d09f669ba0208a66b2b56a1f20c13b4c7eab2dba/src/commands/createStaticWebApp/BuildPresetListStep.ts#L20